sonic-pcied/scripts/pcied

```diff
@@ -41,8 +40,8 @@ class DaemonPcied(DaemonBase):
     def __init__(self, log_identifier):
         super(DaemonPcied, self).__init__(log_identifier)
 
-        platform = get_platform()
-        pciefilePath = os.path.join(PLATFORM_ROOT_PATH, platform, "plugins", PCIE_CONF_FILE)
+        (platform_path, hwsku_path) = device_info.get_paths_to_platform_and_hwsku_dirs()
```

Comment:
Since hwsku_path is not used here, suggest replacing it with `_`. E.g., 

```
(platform_path, _) = device_info.get_paths_to_platform_and_hwsku_dirs()
```